pkg/sources/git/git.go

```diff
@@ -550,7 +550,7 @@ func (s *Git) ScanCommits(ctx context.Context, repo *git.Repository, path string
 
 	gitDir := filepath.Join(path, gitDirName)
 
-	logger.V(1).Info("scanning repo", logValues...)
+	logger.Info("scanning repo", logValues...)
```

Comment:
question: Would it be worth distinguishing this log from the one below, or do we not really care?

Reply:
I'm not really sure what the difference is so I'm going to leave it as is.